### ISSUE TITLE
GetSuperblockMessage to be used when syncing

### ISSUE THREAD

**Issue by girazoki** [witnet/witnet-rust#1544]
The idea is that with the last consolidated superblock hash, peers can ask for the superblock associated so as to retrieve the correct committee size

**Comment by tmpolaczyk** [comment]
In PR #1527, the superblocks are indexed by superblock_index, so if we decide to use the same inventory protocol as the one we use for blocks and transcations, the `InventoryEntry` struct should be modified as:

```rust
pub enum InventoryEntry {
    Tx(Hash),
    Block(Hash),
    SuperBlock(u32),
}
```

**Comment by tmpolaczyk** [comment]
Implemented in #1557
